sst.config.ts

```diff
@@ -73,3 +80,28 @@ export default $config({
     };
   },
 });
+
+function applyMigrations(uri: string) {
+  console.log(`apply migrations to `, uri);
+  execSync(`npx pg-migrations apply --directory ./db/migrations`, {
+    env: {
+      ...process.env,
+      DATABASE_URL: uri,
+    },
+  });
+}
+
+function loadData(uri: string) {
+  console.log("something weird is happening");
+  try {
+    execSync(`pnpm run db:load-data`, {
```

Comment:
if we run this every time, we'll want to check then in load-data if there's enough users already